tests/unit/components/sidebar-menu/SidebarMenu.spec.jsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import { expect, vi } from 'vitest'
+import SidebarMenu from '~/components/sidebar-menu/SidebarMenu'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import NotificationsIcon from '@mui/icons-material/Notifications'
+import GppGoodIcon from '@mui/icons-material/GppGood'
+
+const iconId = 'accordion-icon'
+
+const tabsData = {
+  tab1: {
+    title: 'Tab 1',
+    content: <div>Tab Content1</div>,
+    icon: <GppGoodIcon data-testid={iconId} />
+  },
+  tab2: {
+    title: 'Tab 2',
+    content: <div>Tab Content2</div>,
+    icon: <NotificationsIcon data-testid={iconId} />
+  },
+  tab3: {
+    title: 'Tab 3',
+    content: <div>Tab Content3</div>,
+    icon: <AccountCircleIcon data-testid={iconId} />
+  }
+}
+const handleClick = vi.fn()
+
+describe('SidebarMenu', () => {
+  it('renders without crashing', () => {
+    render(<SidebarMenu handleClick={handleClick} tabsData={tabsData} />)
+  })
+
+  it('renders correct number of menu items', () => {
+    render(<SidebarMenu handleClick={handleClick} tabsData={tabsData} />)
```

Comment:
instead of render 

`<SidebarMenu handleClick={handleClick} tabsData={tabsData} />`

 in every test case, you can use `beforeEach` block. Look at another test files to see how it should be implemented

Reply:
Thank you for comment. Fixed